hackathon_site/hardware/views.py

```diff
@@ -43,6 +44,26 @@ def get(self, request, *args, **kwargs):
         return self.retrieve(request, *args, **kwargs)
 
 
+class IncidentListView(mixins.ListModelMixin, generics.GenericAPIView):
```

Comment:
Once #308 is merged, up to you if you want to add permissions into the view and tests now or if you want to do that as a separate PR

Reply:
I will make a separate PR to add the permissions to these views and tests when IEEE-69 is merged.